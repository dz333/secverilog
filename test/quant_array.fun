(define-fun LH_ARRAY ((x (Array Int Int)) (y Int)) Label
	    (LH (select x y))
)