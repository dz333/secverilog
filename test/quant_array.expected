unsat
unsat
unsat
unsat
Start base conditions are satisfiable? (should be sat)
sat
End base conditions are satisfiable?
unsat
unsat
unknown
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unsat
unknown
